commons/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
     <uses-permission android:name="android.permission.READ_SYNC_STATS"/>
     <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
```

Comment:
Ah, to get file path I guess?
There must be a way to get EXIF information without the file path... but let's keep that as a future optimization: https://github.com/nicolas-raoul/apps-android-commons/issues/32